Library/Homebrew/cmd/help.rb

```diff
@@ -39,4 +39,24 @@ def help
   def help_s
     HOMEBREW_HELP
   end
+
+  def help_cmd(cmd)
```

Comment:
I would prefer this to be called `help_for_command` (more descriptive, no unnecessary abbreviations).